### ISSUE TITLE
[modern tree-sitter] python raw & format string has no "string" scope

### ISSUE THREAD

**Issue by asiloisad** [pulsar-edit/pulsar#575]
### Thanks in advance for your bug report!

- [X] Have you reproduced issue in [safe mode](https://pulsar-edit.dev/docs/atom-archive/faq/sections/what-does-safe-mode-do.html)?
- [X] Have you used the [debugging guide](https://pulsar-edit.dev/docs/atom-archive/hacking-atom/#debugging) to try to resolve the issue?
- [X] Have you checked our [FAQs](https://pulsar-edit.dev/docs/launch-manual/sections/faq/) to make sure your question isn't answered there?
- [X] Have you checked to make sure your issue does not already [exist](https://github.com/issues?q=is%3Aissue+user%3Apulsar-edit+is%3Aopen)?
- [X] Have you checked you are on the latest release of Pulsar?

### What happened?

Python allowed raw string `r'...'`, format string `f'...{...}' and mixed of them `rf'....{...}`. The mixed version has missed "string.format" scope.

```python
r'raw string are ok {11}'
f'format string are ok {11}'
rf'raw and format string are bad {11}'
```

EDIT: The multiline string do not work if `f` or `rf` applied.

### Pulsar version

1.105.2023052201

### Which OS does this happen on?

🪟 Windows

### OS details

10 22H2

### Which CPU architecture are you running this on?

64-bit(x86_64)

### What steps are needed to reproduce this?

1. Open file, set grammar python
2. Type code

### Additional Information:

![image](https://github.com/pulsar-edit/pulsar/assets/29706474/fda113c3-4853-461c-93e0-89f60c3555a0)

![image](https://github.com/pulsar-edit/pulsar/assets/29706474/b97b1f73-470b-4be2-8dbc-74dc6ea4fc44)

**Comment by confused-Techie** [comment]
Thanks a ton for reporting this one!

**Comment by savetheclocktower** [comment]
Pushed a fix for this to #555. I'll keep this issue open until that PR lands.

**Comment by asiloisad** [comment]
I have checked this issue Pulsar 1.106.0 and has not been fixed

**Comment by savetheclocktower** [comment]
PR #555 has the fix. It’ll get landed before the 1.107 release, most likely.
